Lib/multiprocessing/resource_tracker.py

```diff
@@ -296,6 +315,30 @@ def _send(self, cmd, name, rtype):
 getfd = _resource_tracker.getfd
 
 
+def _decode_message(line):
+    if line.startswith(b'{'):
+        try:
+            obj = json.loads(line.decode('ascii'))
+        except Exception as e:
+            raise ValueError("malformed resource_tracker message: %r" % (line,)) from e
```

Comment:
```suggestion
        except Exception as exc:
            raise ValueError("malformed resource_tracker message: %r" % (line,)) from exc
```

Same suggestion for the similar try/except below.

Reply:
I didn't see this message; I merged at the same time you sent it :(

Anyway, I wouldn't change style in a backport.
(Also note that this part of the code was moved -- see `git show --color-moved --color-moved-ws=allow-indentation-change f130b06da` or a fancy Git client.)

Reply:
No problem. It just was a minor suggestion.